config.go

```diff
@@ -495,7 +497,7 @@ type Config struct {
 	// empty if not built with `integration` flag.
 	Dev *lncfg.DevConfig `group:"dev" namespace:"dev"`
 
-	HttpHeaderTimeout time.Duration `long:"http-header-timeout" description:"The maximum duration that the server will wait before timing out reading the headers of an HTTP request."`
+	HTTPHeaderTimeout time.Duration `long:"http-header-timeout" description:"The maximum duration that the server will wait before timing out reading the headers of an HTTP request."`
```

Comment:
also think belongs to the previous commit - we can either move this change to the earlier commit, or we can squash them since they are both small